streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java

```diff
@@ -335,9 +450,37 @@ private void checkProcessorLevelMetrics() {
         checkMetricByName(listMetricProcessor, FORWARD_TOTAL, 18);
     }
 
-    private void checkKeyValueStoreMetricsByType(final String storeType) {
+    private void checkRocksDBMetricsByTag(final String tag) {
+        final List<Metric> listMetricStore = new ArrayList<Metric>(kafkaStreams.metrics().values()).stream()
+            .filter(m -> m.metricName().group().equals("stream-state-metrics") && m.metricName().tags().containsKey(tag))
+            .collect(Collectors.toList());
+        checkMetricByName(listMetricStore, BYTES_WRITTEN_RATE, 1);
```

Comment:
Can we also check the actual value (right now we just check it is not null), for example when we've written messages to two segments, check that the bytes written is the sum of two segments?